content/assignments/mini-libc/tests/io/stat.c

```diff
@@ -0,0 +1,106 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
+#include <stdlib.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <errno.h>
+#include <internal/types.h>
+#include <sys/stat.h>
+#include <string.h>
+
+#define EXISTENT_FILE	"./existent_file"
+
+static char *itoa(long i, char *b)
+{
+	char const digit[] = "0123456789";
+	char *p = b;
+	int shifter = i;
```

Comment:
This cast from long to int generates a segmentation fault, same for the function in fstat.c

Reply:
this is also an issue I can confirm

Reply:
Fixed in [this commit](https://github.com/open-education-hub/operating-systems/pull/209/commits/7eb352bf031956b5aee0efdc3aa3f99b68584a5b). @JunisvaultCo, @vladtf, I think you were the ones reporting this. Please provide your full names and e-mail addresses, so I can add you as co-authors.

Reply:
Plăiașu Iulia-Silvia isplaiasu@gmail.com